### ISSUE TITLE
Implement market price history using eth_getFilterLogs RPC interface

### ISSUE THREAD

**Issue by scottzer0** [AugurProject/augur.js#4]
There's several RPC methods around the logs and filtering, but i think this is the one that will allow us to get price history for markets.

The interface I expect would look something like:

Augur.getMarketPriceHistory(_marketID_, _outcomeID_)

returning a list of objects like:
[...{price: _BigNumber_, timestamp: _moment_ or _Date_}...]

Joey has implemented this in logs.  See https://github.com/AugurProject/augur-core/issues/8

**Comment by scottzer0** [comment]
https://github.com/AugurProject/augur-core/issues/8

**Comment by scottzer0** [comment]
instead of timestamp, you can just provide the block number.

**Comment by tinybike** [comment]
Implemented in c8d3362f5938d836a22f2049e6e72e798a74233e.  Returns an array of objects with `price` and `blockNumber` fields.  Example usage:
```javascript
> Augur.getMarketPriceHistory("0xe7504be05906da9d51eaa4e1ad7c1f29fdd1509c218818793928ddc90303145f", "2")
[ { price: '0.5394656947275467589', blockNumber: '0x17e61' },
  { price: '0.5054586271718324277', blockNumber: '0x17e61' } ]
```
(As with everything else, these values are BigNumbers if `Augur.BigNumberOnly = true`.)

If you provide a callback function as the third argument, `getMarketPriceHistory` is asynchronous.  Async is highly recommended in this case because this method takes a bit of time to run.
